tools/analysis_tools/benchmark.py

```diff
@@ -66,7 +72,7 @@ def measure_inferense_speed(cfg, checkpoint, max_iter, log_interval,
     data_loader = build_dataloader(
         dataset,
         samples_per_gpu=1,
-        workers_per_gpu=cfg.data.workers_per_gpu,
+        workers_per_gpu=0,
```

Comment:
Why set workers to 0?

Reply:
The test found that when it is not set to 0, the test time will increase and become unstable. And the comparison found that detectron2 is forced to be set to 0.

Reply:
Better to add some comments in the code.